app/controllers/story.controller.js

```diff
@@ -0,0 +1,202 @@
+const db = require("../models");
+
+const Story = db.story;
+
+
+// Create and Save a new Story
+
+exports.create = async (req, res) => {
+
+    // Validate request
+    if (req.body.title === undefined) {
+        const error = new Error("Title cannot be empty for story!");
+        error.statusCode = 400;
+        res.status(400).send({
+            message:
+                error.message || "Some error occurred while creating story.",
+        });
+    }
+    // Create a Story
+    const story = {
+        title: req.body.title,
+        description: req.body.description ? req.body.description : "",
+        published: req.body.published ? req.body.published : false,
+        genre: req.body.genre,
+        userId: req.body.userId,
+    };
+
+    // Save Story in the database
+
+    try {
+        const data = await Story.create(story);
+        res.send(data);
+    } catch (err) {
+        res.status(500).send({
+            message:
+                err.message || "Some error occurred while creating the story.",
+        });
+    }  
+}
+
+// Retrieve all Stories from the database.
+
+
+exports.findAll = async (req, res) => {
+    try {
+        const data = await Story.findAll({include : {model: db.user, as :"user" , attributes: ['id', 'firstname', "lastName"]}});
```

Comment:
I'm confused by the `attributes` here, does a story have `firstname` and `lastName` attributes?